File: 프로그래머스/unrated/181890. 왼쪽 오른쪽/왼쪽 오른쪽.py
def solution(str_list):
    answer = []
    l_index = 21
    r_index = 21
    if 'l' in str_list:
        l_index = str_list.index('l')    
    if 'r' in str_list:
        r_index = str_list.index('r')
    
    if l_index < r_index:
        answer = str_list[:l_index]
    else:
        answer = str_list[r_index+1:]
    return answer

File: 프로그래머스/unrated/181890. 왼쪽 오른쪽/왼쪽 오른쪽.js
function solution(str_list) {
    if(str_list.includes('l') || str_list.includes('r')){
        for(let i=0;i<str_list.length;i++){
            if(str_list[i]==='l'){
                str_list.splice(i);
                return str_list;
            }else if(str_list[i]==='r'){
                let result = str_list.splice(i+1);
                return result;
            }
        }
    }else{
        return [];
    }
    
}